ui/round/src/ctrl.ts

```diff
@@ -843,7 +836,8 @@ export default class RoundController {
   private delayedInit = () => {
     const d = this.data;
     if (this.isPlaying() && game.nbMoves(d, d.player.color) === 0 && !this.isSimulHost()) {
-      lichess.sound.play('genericNotify');
+      // lichess.sound.play('genericNotify');
+      // we do this before redirecting for autoplay reasons
```

Comment:
this `if` seems to be empty?

Reply:
Reminds me to test whether simul host gets beeped to hell and back by this change.

Reply:
seems ok, simul host only gets attention switching games on pentatonic - i believe that's the way it was previously.